pkg/operator/operator.go

```diff
@@ -90,32 +98,52 @@ type Operator struct {
 }
 
 func NewOperator(ctx context.Context, operator *operator.Operator) (context.Context, *Operator) {
-	config := &aws.Config{
-		STSRegionalEndpoint: endpoints.RegionalSTSEndpoint,
+	cfg, err := config.LoadDefaultConfig(ctx,
+		config.WithSTSRegionalEndpoint(endpoints.RegionalSTSEndpoint),
+		config.WithRetryer(func() aws.Retryer {
+			return retry.NewStandard(func(o *retry.StandardOptions) {
+				o.MaxAttempts = 5
+			}),
+		}),
+		config.WithAPIOptions(
+			middleware.AddUserAgentKey("CustomUserAgent"),
+			prometheusv1.WithPrometheusMetrics(crmetrics.Registry),
```

Comment:
```suggestion
			prometheusv2.WithPrometheusMetrics(crmetrics.Registry),
```

This needs to be bumped to v2 -- this isn't going to work with the AWS SDK v2